book/src/error.md

```diff
@@ -93,16 +93,20 @@ recovered on the release build in principle.
 That said, in some cases, `panic!` happens from the code out of your control.
 For example, if it is thrown by some of the dependency crates, there's litte you
 can do. You should report the author about the problem, but it's not always the
-behavior is fixed immediately and the fixed version is published.
+behavior is fixed immediately and the fixed version is published. Also, keep in
+mind that depending on what originates the error, some authors can deliberately
+prefer to use `panic!` instead of `Result`.
+Note that `panic!` also happens in rust std library in situations such as division
+by zero or out-of-bounds error when indexing a `Vec`.
 
 In such cases, you can change the following setting included in the template
-`Cargo.toml` generated by `savvy-cli init`. You can set this to `panic =
-"unwind"` to gracefully convert a panic into an R error just like the debug
-build. Note that, the backtrace is not available on the release build because
+`Cargo.toml` generated by `savvy-cli init`. Set this to `panic = "unwind"`
+to gracefully convert a panic into an R error just like the debug build.
+Note that the backtrace is not available on the release build because
 there's no debug info.
 
 ```toml
 [profile.release]
 # ...snip...
-panic = "abort"
+panic = "unwind"
```

Comment:
The intention was to point the location where the user needs to change, so I showed the line before the change, but I'm fine either.